app/models/account.rb

```diff
@@ -0,0 +1,14 @@
+class Account < ApplicationRecord
+  belongs_to :group
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
+
+  has_many :operations
+  has_many :account_credit_cards
+  has_many :account_debits
+  has_many :account_objectives
+
+  validates :name, presence: true
+  validates :kind, presence: true
+
+  enum kind: %i[cash savings cheking_account creditcard investment]
```

Comment:
Hey @wfbraga,

Take a look at the [task](https://trello.com/c/eCQftHiF/10-criar-modelo-account), let's have these kinds: `money, debit, creditcard, objective`.

_Importante note:_ `money` is the only kind without detail (another table with more info).